### DISCUSSION TITLE
Proposta

### DISCUSSION THREAD

**Discussion by edalcin** [sibbr/DarwinCoreBrasil#75] [General]
Obrigado pelo convite Clara!

Acho que poderíamos ter "Classes e Termos", ao invés de só "Termos", como fala no README. Creio ainda que os vocabulários poderiam estar em uma {organização, pasta, documento} a parte, como o "domínio" de alguns termos.

Talvez pensar na estrutura deste repositório, já que está sendo iniciado, seria uma boa ação. Que acha?

**Comment by kguidonimartins** [comment]
> Talvez pensar na estrutura deste repositório, já que está sendo iniciado, seria uma boa ação. Que acha?

Sem dúvida seria ótimo. Tem alguma sugestão de estrutura para o repositório?

```
.
├── docs
├── LICENSE
├── README.md
└── tools
```

Além da pasta de documentos, talvez vamos precisar de uma pasta com ferramentas para algumas automações (atualização da tabela do README, por exemplo).

**Reply by edalcin** [reply]
Acho que está ótimo @kguidonimartins . Talvez uma pasta para {referências, links etc} também.
